src/client/utils/localRequest.js

```diff
@@ -0,0 +1,43 @@
+const PORT = import.meta.env.VITE_PORT;
+const BASE_PATH = `http://localhost:${PORT}/`;
+
+/**
+ * @param {String} endpoint
+ * @param {{}} payload
+ * @returns {Promise<Object>}
+ */
+async function postUrl(endpoint, payload) {
```

Comment:
really small nitpick, but I'm not quite sure what the `url` portion of the function name gives us. I think just calling the function `post` would be better